package.json

```diff
@@ -47,54 +43,33 @@
     "node": ">=8.0.0",
     "npm": ">=5.0.0"
   },
-  "jest": {
-    "testEnvironment": "node",
-    "collectCoverage": true,
-    "coveragePathIgnorePatterns": [
-      "/node_modules/",
-      "/test/fixture"
-    ],
-    "forceExit": true
-  },
   "dependencies": {
     "consola": "^2.4.0",
     "glob-all": "^3.1.0",
     "sass-resources-loader": "^2.0.0"
   },
   "devDependencies": {
-    "@commitlint/cli": "^7.5.0",
-    "@commitlint/config-conventional": "^7.5.0",
+    "@babel/core": "latest",
+    "@babel/preset-env": "latest",
+    "@commitlint/cli": "latest",
+    "@commitlint/config-conventional": "latest",
     "@nuxt/config": "^2.8.1",
-    "@nuxtjs/eslint-config": "^0.0.1",
-    "babel-eslint": "^10.0.1",
-    "codecov": "^3.1.0",
-    "eslint": "^5.13.0",
-    "eslint-config-standard": "^12.0.0",
-    "eslint-plugin-import": "^2.16.0",
-    "eslint-plugin-jest": "^22.2.2",
-    "eslint-plugin-node": "^8.0.1",
-    "eslint-plugin-promise": "^4.0.1",
-    "eslint-plugin-standard": "^4.0.0",
-    "eslint-plugin-vue": "^5.1.0",
-    "get-port": "^4.1.0",
-    "husky": "^1.3.1",
-    "jest": "^24.1.0",
-    "jsdom": "^13.2.0",
+    "@nuxtjs/eslint-config": "latest",
+    "babel-eslint": "latest",
+    "babel-jest": "latest",
```

Comment:
No need `babel-eslint` and `babel-jest`

Reply:
if remove `babel-eslint`, `npm run lint` has errors.

```
/****/src/github.com/aaharu/style-resources-module/lib/module.js
  0:0  error  Parsing error: Cannot find module 'babel-eslint'
Require stack:
- /****/src/github.com/aaharu/style-resources-module/node_modules/vue-eslint-parser/index.js
- /****/src/github.com/aaharu/style-resources-module/node_modules/eslint-plugin-vue/lib/utils/index.js
- /****/src/github.com/aaharu/style-resources-module/node_modules/eslint-plugin-vue/lib/rules/array-bracket-spacing.js
- /****/src/github.com/aaharu/style-resources-module/node_modules/eslint-plugin-vue/lib/index.js
- /****/src/github.com/aaharu/style-resources-module/node_modules/eslint/lib/cli-engine/config-array-factory.js
- /****/src/github.com/aaharu/style-resources-module/node_modules/eslint/lib/cli-engine/cascading-config-array-factory.js
- /****/src/github.com/aaharu/style-resources-module/node_modules/eslint/lib/cli-engine/cli-engine.js
- /****/src/github.com/aaharu/style-resources-module/node_modules/eslint/lib/cli-engine/index.js
- /****/src/github.com/aaharu/style-resources-module/node_modules/eslint/lib/cli.js
- /****/src/github.com/aaharu/style-resources-module/node_modules/eslint/bin/eslint.js

```

Reply:
You need to update `.eslintrc.js` to
```js
module.exports = {
  root: true,
  extends: [
    '@nuxtjs'
  ]
}
```